### PR TITLE
feat: ai nft generator

### PR DESCRIPTION
**Pull request by tim-sean**
- support use AI to generate image for create NFT

### CODE REVIEW

File: src/pagesComponents/CreateItemAI/index.tsx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        onConfirm={(items: string[]) => {
+          setAlbumOpen(false);
+        }}
+        onCancel={() => {
+          setAlbumOpen(false);
+        }}
+      />
+      {/* <Album
+        isModalOpen={true}
+        onConfirm={(items: string[]) => {
```
Comment (ls-bob) [inline src/pagesComponents/CreateItemAI/index.tsx:179]: delete it

File: src/pagesComponents/CreateItemAI/index.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        <ImagePreview />
+        {collection && createType === 'single' && (
+          <CollectionTag
+            nftName={nftName}
+            collectionName={collection.tokenName}
+            src={collection.logoImage}
+            id={tokenId}
+          />
+        )}
+        {/* <div className=" leading-[24px] w-[343px]  mdl:w-[480px] text-[16px] text-[var(--text-secondary)] mt-[16px]">
```
Comment (ls-bob) [inline src/pagesComponents/CreateItemAI/index.tsx:108]: delete it

File: src/pagesComponents/CreateItemByAI/components/Album/index.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+const Album = (props: AlbumModalProps) => {
+  const { infoState } = useGetState();
+  const { isSmallScreen } = infoState;
+  const { isModalOpen, items, onConfirm, onCancel } = props;
+
+  const [rows, setRows] = useState<IAIImage[][]>([]);
+  const [selectedImages, setSelectedImages] = useState<IAIImage[]>([]);
+  const [tipOpen, setTipOpen] = useState(false);
+
+  console.log('-----rows:', rows);
```
Comment (ls-bob) [inline src/pagesComponents/CreateItemByAI/components/Album/index.tsx:31]: delete it

File: src/pagesComponents/CreateItemByAI/components/NegativePrompt/index.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import { InputTextAreaProps } from 'aelf-design';
+
+export function NegativePrompt(props: InputTextAreaProps) {
+  return (
+    <Input.TextArea
+      className={styles['text-prompt']}
+      maxLength={400}
+      showCount={{
+        formatter: ({ value, count, maxLength }) => {
+          console.log('value', value);
```
Comment (ls-bob) [inline src/pagesComponents/CreateItemByAI/components/NegativePrompt/index.tsx:12]: delete it

File: src/pagesComponents/CreateItemAI/index.tsx (hunk 1 above)
Comment (tim-sean) [reply src/pagesComponents/CreateItemAI/index.tsx]: this file is no longer in use,  has deleted

**Pull request merged**